python/fbprophet/forecaster.py

```diff
@@ -100,6 +105,7 @@ def __init__(
             weekly_seasonality='auto',
             daily_seasonality='auto',
             holidays=None,
+            append_holidays=None,
```

Comment:
This also needs to be added to the prophet_copy function here: https://github.com/facebook/prophet/blob/master/python/fbprophet/diagnostics.py#L161